routers/issueRouter.js

```diff
@@ -42,4 +42,16 @@ router.post( '/', jsonParser, function( req, res ){
 	});
 });
 
-module.exports = router;
+// PUT issue, updating a current issue
+router.put( '/', jsonParser, function( req, res){
+	var issue = req.body;
+	db.findByIdAndUpdate(issue._id, { name: issue.name, description: issue.description, location: { lat: issue.location[lat], long: issue.location[long]}, priority: issue.priority, open: issue.open, reportedDate: issue.reportedDate, resolvedDate: issue.resolvedDate}, function( err, doc ){
```

Comment:
The clients should be abiding by the structure of the schema object anyway, so I'm imagining the put request body to be something like:
    `{ _id: some_id, attribute_to_change: new_value, another_attribute_to_change: another_new_value, ...}`

I don't think it needs the entire object though, just the changed keys. At least that's how I'm understanding it from their documentation (that isn't all that detailed). I'll test it extensively later to make sure it works.